staging/src/k8s.io/legacy-cloud-providers/azure/clients/vmssclient/azure_vmssclient.go

```diff
@@ -234,6 +234,20 @@ func (c *Client) CreateOrUpdate(ctx context.Context, resourceGroupName string, V
 	return nil
 }
 
+// CreateOrUpdateSync sends the request to arm client and DO NOT wait for the response
+func (c *Client) CreateOrUpdateSync(ctx context.Context, resourceGroupName string, VMScaleSetName string, parameters compute.VirtualMachineScaleSet) (*azure.Future, *retry.Error) {
```

Comment:
could you add ratelimiter and throttle handling? you can refer `CreateOrUpdate()` to do this

Reply:
I'm on it.

Reply:
And let's rename the function to `CreateOrUpdateAsync()`, since it returns a future instead of waiting for it completes